### ISSUE TITLE
plugin exception with last updated pycharm

### ISSUE THREAD

**Issue by gbonline** [42Crunch/intellij-openapi-editor#27]
pycharm info:
```
PyCharm 2022.3 (Community Edition)
Build #PC-223.7571.203, built on November 30, 2022
Runtime version: 17.0.5+1-b653.14 x86_64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
macOS 12.6.1
GC: G1 Young Generation, G1 Old Generation
Memory: 2048M
Cores: 8
Metal Rendering is ON
Non-Bundled Plugins:
    com.mroche.JsonHelper (1.2.0)
    com.crunch42.openapi (1.41)
    DBN (3.3.3100.0)
```

plugin version: 1.41
exception:

```
java.lang.NoSuchMethodError: 'void org.snakeyaml.engine.v2.serializer.Serializer.open()'
	at com.xliic.openapi.parser.ast.node.Node.dump(Node.java:231)
	at com.xliic.openapi.quickfix.QuickFix.getInstance(QuickFix.java:60)
	at com.xliic.openapi.services.QuickFixService.process(QuickFixService.java:159)
	at com.xliic.openapi.services.QuickFixService.loadJsonConfig(QuickFixService.java:221)
	at com.xliic.openapi.services.QuickFixService.load(QuickFixService.java:59)
	at com.xliic.openapi.OpenAPIStartupActivity.runActivity(OpenAPIStartupActivity.java:45)
	at com.intellij.ide.startup.impl.StartupManagerImpl.runActivityAndMeasureDuration(StartupManagerImpl.kt:340)
	at com.intellij.ide.startup.impl.StartupManagerImpl.access$runActivityAndMeasureDuration(StartupManagerImpl.kt:66)
	at com.intellij.ide.startup.impl.StartupManagerImpl$runPostStartupActivities$2$2$1.invoke(StartupManagerImpl.kt:279)
	at com.intellij.ide.startup.impl.StartupManagerImpl$runPostStartupActivities$2$2$1.invoke(StartupManagerImpl.kt:278)
	at com.intellij.openapi.progress.CancellationKt.withCurrentJob$lambda-0(cancellation.kt:17)
	at com.intellij.openapi.progress.Cancellation.withCurrentJob(Cancellation.java:60)
	at com.intellij.openapi.progress.CancellationKt.withCurrentJob(cancellation.kt:17)
	at com.intellij.openapi.progress.CoroutinesKt.blockingContext(coroutines.kt:142)
	at com.intellij.ide.startup.impl.StartupManagerImpl.runPostStartupActivities(StartupManagerImpl.kt:278)
	at com.intellij.ide.startup.impl.StartupManagerImpl.access$runPostStartupActivities(StartupManagerImpl.kt:66)
	at com.intellij.ide.startup.impl.StartupManagerImpl$runStartupActivities$2.invokeSuspend(StartupManagerImpl.kt:184)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106)
	at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:570)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.executeTask(CoroutineScheduler.kt:750)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runWorker(CoroutineScheduler.kt:677)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:664)

```

**Comment by ak1394** [comment]
Thanks for reporting, this is a same issue as #26 we're working on the fix.

**Comment by ak1394** [comment]
This issue has been resolved in 1.46 of the extension.
